std/traits.d

```diff
@@ -4815,6 +4823,21 @@ template ImplicitConversionTargets(T)
         alias ImplicitConversionTargets = AliasSeq!();
 }
 
+///
+@safe unittest
+{
+    import std.meta : AliasSeq;
+
+    static assert(is(ImplicitConversionTargets!(ulong) == AliasSeq!(float, double, real)));
+    static assert(is(ImplicitConversionTargets!(string) == AliasSeq!(const(char)[])));
```

Comment:
This function is pretty though to understand.
How about adding a few more examples?

```diff
diff --git a/std/traits.d b/std/traits.d
index 997ce3dcc..66236cb22 100644
--- a/std/traits.d
+++ b/std/traits.d
@@ -4830,6 +4830,16 @@ template ImplicitConversionTargets(T)
 
     static assert(is(ImplicitConversionTargets!(ulong) == AliasSeq!(float, double, real)));
     static assert(is(ImplicitConversionTargets!(string) == AliasSeq!(const(char)[])));
+    static assert(is(ImplicitConversionTargets!(int) == AliasSeq!(long, ulong, float, double, real)));
+
+    static assert(is(ImplicitConversionTargets!(float) == AliasSeq!(double, real)));
+    static assert(is(ImplicitConversionTargets!(double) == AliasSeq!(real)));
+
+    static assert(is(ImplicitConversionTargets!(char) == AliasSeq!(wchar, dchar, byte, ubyte, short, ushort, int, uint, long, ulong, float, double, real)));
+    static assert(is(ImplicitConversionTargets!(wchar) == AliasSeq!(dchar, short, ushort, int, uint, long, ulong, float, double, real)));
+    static assert(is(ImplicitConversionTargets!(dchar) == AliasSeq!(int, uint, long, ulong, float, double, real)));
+
+    static assert(is(ImplicitConversionTargets!(void*) == AliasSeq!(void*)));
 
     interface A {}
     interface B {}
```

Though the downside is that it depends on CentTypeList which is currently empty, but might become cent, ucent one day (I think LDC is making progress on that one).